src/commands/data/schemas/init.ts

```diff
@@ -0,0 +1,53 @@
+import { writeFile, mkdir } from 'fs/promises';
+import * as osPath from 'path';
+import { epilogue } from '../../../helpers/util';
+
+export const command = 'init <name>';
+export const desc = 'Create a basic schema configuration file';
+export const builder = (yargs: any) => epilogue(yargs)
+  .positional('name', {
+    demandOption: true,
+    description: 'The name of the new schema',
+    type: 'string',
+  })
+  .options({
+    path: {
+      description: 'The path to the folder where the schema should be created',
+      type: 'string',
+      default: './schemas',
+    },
+  });
+
+export const handler = async function init({ name, path }: { name: string; path: string; }) {
+  await mkdir(path, { recursive: true });
+  const filePath = osPath.join(path, `${name}.json`);
+
+  await writeFile(filePath, JSON.stringify(createSchema(name), null, 2));
+
+  console.log(`✅  Successfully created ${filePath}`);
+};
+
+function createSchema(name: string) {
+  return {
+    $schema: 'https://swagger.extrahorizon.com/cli/1.10.0/config-json-schemas/Schema.json',
```

Comment:
Potential follow up PR: Would like to make a helper to read the package.json and get the version from there